core/src/main/scala/ai/lum/odinson/OdinsonIndexWriter.scala

```diff
@@ -280,23 +284,26 @@ object OdinsonIndexWriter {
         val vocab = Vocabulary.fromDirectory(dir)
         (dir, vocab)
     }
-    // Always store the display field, also store these additional fields
-    val settings = IndexSettings(Seq(displayField) ++ storedFields)
     new OdinsonIndexWriter(
-      directory,
-      vocabulary,
-      settings,
-      documentIdField,
-      sentenceIdField,
-      sentenceLengthField,
-      normalizedTokenField,
-      addToNormalizedField.toSet,
-      incomingTokenField,
-      outgoingTokenField,
-      sortedDocValuesFieldMaxSize,
-      maxNumberOfTokensPerSentence,
-      invalidCharacterReplacement,
-      displayField
+      // format: off
+      directory            = directory, 
+      vocabulary           = vocabulary,
+      // Always store the display field, also store these additional fields
+      settings             = IndexSettings(Seq(displayField) ++ storedFields),
+      documentIdField      = config[String]("odinson.index.documentIdField"),
```

Comment:
This is wrong I think -- when you make the new IndexWriter you're overriding whatever you passed in

Reply:
Can you clarify?

Reply:
So the the `displayField` is not necessarily stored anymore? 😬

Reply:
no -- but there's a check for it, it will throw an exception if you don't have it: 
https://github.com/lum-ai/odinson/blob/master/core/src/main/scala/ai/lum/odinson/OdinsonIndexWriter.scala#L285